bridge/github/config.go

```diff
@@ -141,7 +124,6 @@ func (g *Github) Configure(repo repository.RepoCommon, params core.BridgeParams)
 	}
 
 	conf[core.ConfigKeyTarget] = target
-	conf[core.ConfigKeyTokenId] = tokenObj.ID().String()
```

Comment:
If my understanding is correct bridges doesn't store the auth id anymore ? git-bug is responsible of finding the user token before the import and export operations

Reply:
correct